src/components/section/aside.jsx

```
@@ -0,0 +1,118 @@
+                        <li key={key}>
+                            <a onClick={() => setOpenModal(true)}>
+                                <span>{chatbot.icon} {chatbot.title}</span>
+                            </a>
+                        </li>
+                    ))}
+                </ul>
+                {openModal && (
+                    <div className="chatbot-overlay">
+                        <div className="chatbot-content" onClick={(e) => e.stopPropagation()}>
+                            <Chatbot />
+                            <button onClick={() => setOpenModal(false)}><MdClear /></button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)